app/controllers/v1/nod_callbacks_controller.rb

```diff
@@ -7,65 +7,84 @@ class NodCallbacksController < ApplicationController
     include ActionController::HttpAuthentication::Token::ControllerMethods
     include DecisionReviewV1::Appeals::LoggingUtils
 
-    service_tag 'nod-callbacks'
+    service_tag 'appeal-application'
 
     skip_before_action :verify_authenticity_token, only: [:create]
     skip_before_action :authenticate, only: [:create]
     skip_after_action :set_csrf_header, only: [:create]
     before_action :authenticate_header, only: [:create]
 
-    STATUSES_TO_IGNORE = %w[sent delivered temporary-failure].freeze
+    STATSD_KEY_PREFIX = 'api.decision_review.notification_callback'
+
+    DELIVERED_STATUS = 'delivered'
 
     def create
-      return render json: nil, status: :not_found unless Flipper.enabled? :nod_callbacks_endpoint
+      return render json: nil, status: :not_found unless enabled?
 
       payload = JSON.parse(request.body.string)
+      status = payload['status']&.downcase
 
-      log_params = {
-        key: :callbacks,
-        form_id: '10182',
-        user_uuid: nil,
-        upstream_system: 'VANotify',
-        body: payload.merge('to' => '<FILTERED>') # scrub PII from logs
-      }
+      StatsD.increment("#{STATSD_KEY_PREFIX}.received", tags: { status: })
 
-      # save encrypted request body in database table for non-successful notifications
-      payload_status = payload['status']&.downcase
-      if STATUSES_TO_IGNORE.exclude? payload_status
-        begin
-          NodNotification.create!(payload:)
-        rescue ActiveRecord::RecordInvalid => e
-          log_formatted(**log_params.merge(is_success: false), params: { exception_message: e.message })
-          return render json: { message: 'failed' }
-        end
+      if status == DELIVERED_STATUS
+        tags = ['service:supplemental-claim', 'function: form or evidence submission to Lighthouse']
+        StatsD.increment('silent_failure_avoided', tags:)
       end
 
-      log_formatted(**log_params.merge(is_success: true))
+      begin
+        DecisionReviewNotificationAuditLog.create!(notification_id: payload['id'],
+                                                   reference: payload['reference'],
+                                                   status:,
+                                                   payload:)
+      rescue ActiveRecord::RecordInvalid => e
+        log_formatted(**log_params(payload, false), params: { exception_message: e.message })
+        return render json: { message: 'failed' }
+      end
```

Comment:
**Future suggestion**: Use `if .save` on AR create/update instead of catching an exception. It will also make testing easier because you don't need fake an exception or check for exception. 

```ruby
audit_log = DecisionReviewNotificationAuditLog.new(....)

if audit_log.save
  log_formatted(**log_params(payload, true))
  render json: { message: 'success' }
else
  exception_message = "Validation failed: #{audit_log.errors.full_messages.join(', ')}"
  log_formatted(**log_params(payload, false), params: { exception_message:  })
  render json: { message: 'failed' }
end
```

I added the "Validation failed" part to the message to match the ActiveRecord::RecordInvalid message.